docs/management.md

```diff
@@ -418,6 +446,66 @@ management.oauth_provider_url = https://my-web-portal
 Additionally, RabbitMQ also accepts a JWT token in the HTTP `Authorization` header when the user lands on the management UI.
 
+With `sp_initiated` logon types, there is no need to configure the `oauth_provider_url` if `auth_oauth2.issuer` was set. However, for `idp_initiated` flows the `auth_oauth2.issuer` url may not necessarily be the url where to send users to authenticate. When this occurs, the `management.oauth_provider_url` overrides the `auth_oauth2.issuer` url.
+
+### Support multiple OAuth 2.0 resources {#support-multiple-resources}
+
+RabbitMQ 3.13 introduced support for multiple OAuth 2.0 resources in the OAuth 2.0 plugin and in the Management plugin.
+
+Once you have configured the [OAuth 2.0 plugin](./oauth2#multiple-resource-servers) with all the required OAuth 2.0 resources, they are automatically supported by the management UI.
+
+Given the following OAuth 2.0 plugin configuration which consists of four OAuth 2.0 resources:
+```ini
+auth_oauth2.issuer = http://some_idp_url
+auth_oauth2.scope_prefix = rabbitmq.
+auth_oauth2.resource_servers.1.id = rabbit_prod
+auth_oauth2.resource_servers.2.id = rabbit_dev
+auth_oauth2.resource_servers.3.id = rabbit_qa
+auth_oauth2.resource_servers.4.id = rabbit_internal
+```
+
+This is the minimum configuration to enable all resources in the management UI:
+```ini
+management.oauth_enabled = true
+management.oauth_scopes = openid rabbitmq.tag:management rabbitmq.read:*/*
+```
+
+However, it is possible to configure the client_id, the labels for each resource, or the login type, the scopes
+for each resource or the provider_url. Or even disable some resources in the management UI.
+See below an example configuration:
+
+```ini
+management.oauth_enabled = true
+management.oauth_initiated_logon_type = sp_initiated
+management.oauth_scopes = openid rabbitmq.tag:management rabbitmq.read:*/*
+
+management.oauth_resource_servers.1.id = rabbit_prod
+management.oauth_resource_servers.1.oauth_client_id = rabbit_prod_mgt_ui
+management.oauth_resource_servers.1.label = RabbitMQ Production
+
+management.oauth_resource_servers.2.id = rabbit_dev
+management.oauth_resource_servers.2.oauth_client_id = rabbit_dev_mgt_ui
+management.oauth_resource_servers.2.label = RabbitMQ Development
+
+management.oauth_resource_servers.3.id = rabbit_qa
+management.oauth_resource_servers.3.label = RabbitMQ QA
+management.oauth_resource_servers.3.oauth_initiated_logon_type = idp_initiated
+management.oauth_resource_servers.3.oauth_provider_url = http://qa_url
+
+management.oauth_resource_servers.4.id = rabbit_internal
+management.oauth_resource_servers.4.disabled = true
+```
+
+
+This is the Management UI layout for the above configuration with Basic Authentication disabled in the Management UI (`management.oauth_disable_basic_auth = true`).
```

Comment:
How about: This is the Management UI layout for the previous configuration with basic authentication deactivated in the Management UI (`management.oauth_disable_basic_auth = true`).